casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -778,16 +778,21 @@ class RuntimeManagerImpl[F[_]: Concurrent: Metrics: Span: Log](
   def playExploratoryDeploy(term: String, hash: StateHash): F[Seq[Par]] = {
     // Create a deploy with newly created private key
     val (privKey, _) = Secp256k1.newKeyPair
+
+    // Creates signed deploy
     val deploy = ConstructDeploy.sourceDeploy(
       term,
       timestamp = System.currentTimeMillis,
-      phloLimit = Long.MaxValue,
+      // Hardcoded phlogiston limit / 1 REV if phloPrice=1
+      phloLimit = 100 * 1000 * 1000,
```

Comment:
Better makes this configurable

Reply:
I agree. We can do this as part of refactoring RuntimeManager.